trunk/SUAVE/Methods/Flight_Dynamics/Static_Stability/compute_aero_derivatives.py

```diff
@@ -100,15 +123,25 @@ def compute_aero_derivatives(segment, h=1e-4):
     iterate.conditions(perturbed_segment) 
     
     # set segment derivatives based on perturbed segment
-    dV = np.atleast_2d(Vx_plus-velocity_vector[:,0]).T
-    dCL    = perturbed_segment.state.conditions.aerodynamics.lift_coefficient - segment.state.conditions.aerodynamics.lift_coefficient
-    dCD    = perturbed_segment.state.conditions.aerodynamics.drag_coefficient - segment.state.conditions.aerodynamics.drag_coefficient
-
+    dV    = np.atleast_2d(Vx_plus-velocity_vector[:,0]).T
```

Comment:
This is effectively changing the AoA too. So it's not truly decoupled.

Reply:
removed velocity perturbation since it is not necessary for any stability derivatives